src/htmlreporter.ts

```diff
@@ -32,11 +32,13 @@ class HTMLReporter {
   public async generate() {
     try {
       const report = await this.renderTestReport();
-      const outputPath = this.getConfigValue("outputPath") as string;
+      let outputPath = this.getConfigValue("outputPath") as string;
+      // Replace <rootDir> with the actual path to root
+      outputPath = outputPath.replace(/<rootdir>/gi, ".");
```

Comment:
I'm afraid it is a tad more complicated ;) 
For instance if you set the jest `rootDir` config option to something else than the project root.  

See https://github.com/jest-community/jest-junit/blob/master/utils/replaceRootDirInPath.js

Reply:
I agree that this should be the case. 
However this would be considered a new feature, as the 2.x way of handling this was a simple rewrite (and tbf the request was to restore that functionality). 

With that said though, I will research the matter a bit more and update the PR